poi.config.js

```diff
@@ -85,7 +85,9 @@ module.exports = {
   },
   envs: {
     API_PATH: '/api/v1/',
-    AUTH_JWT_SCHEME
+    AUTH_JWT_SCHEME,
+    // The key to store the Vuex state in the storage
+    VUEX_STORAGE_KEY: 'APP_STORE'
```

Comment:
My vote goes for `TAILOR_STATE` or even better `TAILOR_APP_STATE` because:
1. you really want to prefix it with app name cause otherwise it will potentially clash with other locally developed apps running on same port
2. seing state suffix indicates that it came from vuex